include/aspect/parameters.h

```diff
@@ -192,7 +192,7 @@ namespace aspect
      * @{
      */
     bool                           include_melt_transport;
-    double                         melt_transport_threshold;
+
```

Comment:
Oh, who knew that these variables already existed in the current code base :-) I wonder how they got in there!?